Report/Template.Rmd

```diff
@@ -243,17 +251,262 @@ datasets = read.and.clean(dataset = "easySHARE_rel6_0_0.rda", wav = 99)
 ```
 
 \section{Multidisciplinary and crossnational summary statistics}
-JULIAN
-\subsection{Theory and Design}
-\subsection{Implementation}
-\subsection{Empirical Results}
+
+## Theory and Design
+
+Since we are working with a cross-national survey, it is useful to to present an overview on a country-by-country basis. For each country, survey participants will be classified according to their age group. Within each class, shares of gender-specific as well as overall labor participation rates and health indicators are calculated and discussed. 
+
+
+## Implementation
+
+An efficient way of seperating the easySHARE dataset into country, gender and age is explained in the following. In order to minimize hard-coding and possibly related errors, a grid is expanded between named variables.
+Functions that calculate group shares per country and labor participation rates can subsequently be applied onto the defined grid. 
+
+Besides the target data frame, `group.share` takes a boolean variable as an input that specifies whether aggregation on a national level is intended. The share of males or females (per country) is calculated by finding the sum of occurences and dividing by their total, represented by the length of the vector. A similar procedure that allows for multiple subsets and based upon `dplyr` is used in `labor`.
```

Comment:
It's actually a bit different (and perhaps confusing in the code). They're two different functions entirely.
`group.shares` calculates the average or percentage for a given data frame + metric BY COUNTRY. Example: average grip strength for men.  That's why we split the df's into men/women then apply this function (which already calculates after grouping by countries in the `tapply` part).
`labor` calculates labor participation rate. That's why the `expand.grid` function is used, so that we can calculate labor participation rates for all combinations of country, gender, and age. 
`expand.grid` isn't relevant in the context of calculating group.shares

Reply:
So basically `group.shares` is used to calculate sample stats (table 4.1 in the original paper), health indicators (tables 4.2-3 in the og paper), and labor supply choice for m/f (tables 4.6-7)
`labor` calculates tables 4.4 and 4.5

Reply:
holy moly! thanks

Reply:
and integrated